### DISCUSSION TITLE
Terminal is running on release app

### DISCUSSION THREAD

**Discussion by ithustle** [tauri-apps/tauri#10255] [Q&A] [answered]
Hello,

I'm new on Tauri and I'm releasing my first app on it but I realize that Terminal is opening on release mode. Is it supposed to happen this?
Some help will be grateful

**Comment by FabianLars** [comment, marked as answer]
The templates by default have this line in main.rs: https://github.com/tauri-apps/create-tauri-app/blob/dev/templates/_base_/src-tauri/src/main.rs.lte#L1-L2

Did you by any chance remove it?

**Reply by ithustle** [reply]
Yep. My bad. Thanks.
